### PR TITLE
KubeVirt platform troubleshooting documentation

### PR DESCRIPTION
**Pull request by davidvossel**
This PR adds more troubleshooting steps that are specific to the KubeVirt platform.

### CODE REVIEW

**Review by bryan-cox — APPROVED** [review]
/lgtm

File: docs/content/how-to/kubevirt/troubleshooting-kubevirt-cluster.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+* Load Balancers for the guest cluster are by default handled by the kubevirt-cloud-controller-manager within the HCP namespace. Ensure the kccm pod is online and look at the pod’s logs for errors or warnings. The kccm pod can be identified in the HCP namespace with this command `oc get pods -n <hcp namespace> -l app=cloud-controller-manager`
+
+## Guest Cluster PVCs are not available
+
+* Look for events and details associated with the PVC to understand what errors are occurring
+* If the PVC is failing to attach to a Pod, look at the logs for the kubevirt-csi-node daemonset component within the guest to gain further insight into why that might be occurring. The kubevirt-csi-node pods can be identified for each node with this command `oc get pods -n openshift-cluster-csi-drivers -o wide -l app=kubevirt-csi-driver`
+* If the PVC is unable to bind to a PV, look at the logs of the kubevirt-csi-controller component within the HCP namespace. The kubevirt-csi-controller pod can be identified within the hcp namespace with this command `oc get pods -n <hcp namespace> -l app=kubevirt-csi-driver`
```
Comment (davidvossel) [inline docs/content/how-to/kubevirt/troubleshooting-kubevirt-cluster.md:48]: @awels can you take a look at this section and see if it is detailed enough. The goal here is to provide some basic steps someone could go through in order to find the root cause for why guest cluster pvcs are not becoming available or attaching to pods.

**Comment by davidvossel** [comment]
@bryan-cox could you give another lgtm, i updated the doc to revise a few of the cli command examples to be more accurate

**Pull request merged**